### PR TITLE
Update SDK_reference.md

### PR DESCRIPTION
**Pull request by simontrinh0911**

### CODE REVIEW

**Review by Yashwantsinghraakawat — APPROVED** [review]
How it's work?

**Comment by sandipnlakhani** [comment]
I found it back my Pi account with Wallet thanks you very much 👍
Love from India🇮🇳🇮🇳🇮🇳💌💌❣️
@sandipnlakhani

**Comment by Jiggnubs** [comment]
Hello. I've joined but a newbie everyday. Is okay!!!

**Comment by will2473** [comment]
How to work it this?

**Comment by Olumeni1** [comment]
Okay

**Comment by bsjharwal** [comment]
Hii  I AM FROM INDIA HOW MUCH VALUE IN INDAA PI.. AND WHEN SELLL HIS..

**Comment by will2473** [comment]
I don't know what do you say?

**Review by NGUYENDAIBAO — CHANGES_REQUESTED** [review]
``

File: SDK_reference.md [hunk 1]
```diff
@@ -1,242 +1,3 @@
-# Client SDK reference:
```
Comment (Aqeebrao) [inline SDK_reference.md:1]: #291

File: SDK_reference.md [hunk 2]
```diff
@@ -1,242 +1,3 @@
...
-
-Add the following `script` tags to all pages where you need to call the Pi Apps SDK:
-
-```html
-<script src="https://sdk.minepi.com/pi-sdk.js"></script>
-<script>Pi.init({ version: "2.0" })</script>
-```
-
-The config object passed to the init function accepts the following keys:
-* `version` (string, required) - this is required to ensure compatibility of your app with newer SDK versions that might bring
```
Comment (Aqeebrao) [inline SDK_reference.md:13]: # # #295

File: SDK_reference.md [hunk 3]
```diff
@@ -1,242 +1,3 @@
...
-  breaking changes (in which case breaking changes will be implemented under a higher version number)
```
Comment (Aqeebrao) [inline SDK_reference.md:14]: #292

File: SDK_reference.md [hunk 4]
```diff
@@ -1,242 +1,3 @@
...
-* `sandbox`: (boolean, optional) - this enables you to configure the SDK to run in the sandbox.
-
-### Using the SDK in sandbox mode:
```
Comment (Aqeebrao) [inline SDK_reference.md:17]: #294

File: SDK_reference.md [hunk 5]
```diff
@@ -1,242 +1,3 @@
...
-## Authentication
-
-> **Warning:** The user information obtained with this method should not be passed to your backend and should
-> only be used for presentation logic (e.g displaying the user’s username).
-> **On your backend, use the Platform API as the source of truth.** You can verify the user's
-> identity by requesting the /me endpoint from your backend, using the access token obtained with this method.
-
-
-```typescript
-Pi.authenticate(scopes: Array<string>, onIncompletePaymentFound: Function<PaymentDTO>): Promise<AuthResult>
```
Comment (Aqeebrao) [inline SDK_reference.md:44]: #255

File: SDK_reference.md [hunk 6]
```diff
@@ -1,242 +1,3 @@
...
-```
-
-Return value:
-
-```typescript
-type AuthResult = {
```
Comment (Aqeebrao) [inline SDK_reference.md:50]: #255

File: SDK_reference.md [hunk 7]
```diff
@@ -1,242 +1,3 @@
...
-  user: {
-    uid: string,
-    username: string
-  }
-}
-```
-
-### `scopes`
-
-Available scopes: `username`, `payments`.
```
Comment (Aqeebrao) [inline SDK_reference.md:61]: #293

File: SDK_reference.md [hunk 8]
```diff
@@ -1,242 +1,3 @@
...
-```
-
-Return value:
-
-```typescript
-type AuthResult = {
-  accessToken: string,
-  user: {
-    uid: string,
-    username: string
```
Comment (Aqeebrao) [inline SDK_reference.md:54]: #293

File: SDK_reference.md [hunk 9]
```diff
@@ -1,242 +1,3 @@
...
-}
-```
-
-### `scopes`
-
-Available scopes: `username`, `payments`.
-
-> **Not yet implemented**
->
-> Currently, all calls to the `authenticate` method will assume all scopes have been requested - i.e all calls
```
Comment (Aqeebrao) [inline SDK_reference.md:65]: #293

File: SDK_reference.md [hunk 10]
```diff
@@ -1,242 +1,3 @@
...
-> are interpreted as though the first argument were `['username', 'payments']`.
-> However, you should implement your app by only adding the scopes you need when calling `authenticate`.
```
Comment (Aqeebrao) [inline SDK_reference.md:67]: #293

File: SDK_reference.md [hunk 11]
```diff
@@ -1,242 +1,3 @@
...
-> Scopes support will be added before the general release of the Pi platform.
-
-Here is a breakdown of various keys available on the `AuthResult['user']` object, and the scopes required for those keys
```
Comment (Aqeebrao) [inline SDK_reference.md:70]: #255

**Review by Aqeebrao — APPROVED** [review]

**Comment by MinerDaveEverist6758** [comment]
Have trouble logging in

File: SDK_reference.md (hunk 1 above)
Comment (Osa717) [inline SDK_reference.md:1]: Hi

**Review by Osa717 — APPROVED** [review]

File: SDK_reference.md (hunk 1 above)
Comment (Olumeni1) [reply SDK_reference.md:1]: Hello how are you doing? And I hope work is going well, I will be waiting

**Comment by Olumeni1** [comment]
Happy returns 🎉

**Comment by will2473** [comment]
Hello.. 🤗

**Comment by Olumeni1** [comment]
Hello write to me only on my Gmail wisdomolumeni32@gmail.com

**Comment by Olumeni1** [comment]
Don't follow without contacting me on Gmail wisdomolumeni32@gmail.com or my call line @+2349019760402

**Comment by Aqeebrao** [comment]
#255

**Comment by Aqeebrao** [comment]
#292

**Comment by Aqeebrao** [comment]
#285

**Comment by Aqeebrao** [comment]
#233

**Review by adizioloo — COMMENTED** [review]
@adizioloo

**Comment by Olumeni1** [comment]
Olumeni1

**Comment by Olumeni1** [comment]
I'm the real hero. Call me on 09019760402.

File: SDK_reference.md (hunk 1 above)
Comment (Olumeni1) [reply SDK_reference.md:1]: > Hi

What are you waiting for, my account is not safe so neglect any massage or task or change for now. Thank you all for understanding and cooperation, do not accept any changes so work with our first agreement

**Review by Olumeni1 — COMMENTED** [review]
What are you waiting for, my account is not safe so neglect any massage or task or change for now. Thank you all for understanding and cooperation, do not accept any changes so work with our first agreement

**Comment by Olumeni1** [comment]
What are you waiting for, my account is not safe so neglect any massage or task or change for now. Thank you all for understanding and cooperation, do not accept any changes so work with our first agreement

**Comment by Olumeni1** [comment]
Avoid clicking on any link,
What are you waiting for, my account is not safe so neglect any massage or task or change for now. Thank you all for understanding and cooperation, do not accept any changes so work with our first agreement

**Comment by 301166** [comment]
I dont find accept law page?!

**Pull request closed without merging**